maths/find_max.rb

```diff
@@ -0,0 +1,66 @@
+# A ruby program to find max from a set of elements
+
+module FindMax
+  # This find_max method will return the max element out of the array
+  def self.find_max(*array)
+    max = array[0]
+    array.each do |a|
+      if a >= max
+        max = a
+      end
+    end
+    puts "The Max of the following elements #{array} is #{max}."
```

Comment:
Can I suggest to have your methods to just return the string rather than `puts`? We can use `puts` down below where you call your method to see the result.